packages/pointer_interceptor/lib/src/web.dart

```diff
@@ -22,11 +22,8 @@ void _registerFactory({bool debug = false}) {
   final String viewType = _getViewType(debug: debug);
   ui.platformViewRegistry.registerViewFactory(viewType, (int viewId) {
     final html.Element htmlElement = html.DivElement()
-      ..style.top = '0'
-      ..style.right = '0'
-      ..style.bottom = '0'
-      ..style.left = '0'
-      ..style.position = 'relative';
+      ..style.width = '100%'
+      ..style.height = '100%';
```

Comment:
Is this change fine for stable as well? (If not, you could always update the Flutter SDK requirement so Flutter 2.2 won't versions that have this change.)

Reply:
Yes, thanks for asking. I tested this both with `stable` and `master`, manually and with the driver tests.

This is backwards compatible, it's just that there's many ways to "cover a parent div" with CSS, but in the engine we only check "are style.width and style.height set (to any value?) else: warn user".

This code is to disable that warning from our own packages.